src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -26,18 +28,18 @@ function getClassName(readOnly, errorText) {
   return styles['read-write-container'];
 }
 
-function CustomFieldInputMultipleChoice(props) {
+const CustomFieldInputMultipleChoice = forwardRef((props, ref) => {
   const autocompleteRef = useRef();
   const [autocompleteValue, setAutocompleteValue] = useState('');
   const [expanded, setExpanded] = useState(false);
   const [value, setValue] = useState(props.value);
-  const visibleChoices = props.choices
-    .filter(choice => choice.label.includes(autocompleteValue))
-    .filter(choice => !value.includes(choice));
+  const [visibleChoices, setVisibleChoices] = useState(getVisibleChoices());
```

Comment:
I don't really see understand visible choices needs to be a stateful property. It looks like this is computed from value and autocompleteValue. Making it stateful seems to just add a bunch of code around managing it.